app/javascript/components/ModalWizard/index.js

```diff
@@ -0,0 +1,9 @@
+import ModalWizard from './ModalWizard';
+import ModalWizardBody from './ModalWizardBody';
+import ModalWizardStateProvider from './ModalWizardStateProvider';
+
+// TODO maybe move these up to patternfly-react?
```

Comment:
potentially ModalWizard can...i think `ModalWizardStateProvider` can be lifted to Redux, but I would like to go ahead with this approach for now to keep us moving...

Reply:
Yeah, sounds good. That was more a long-term idea-- i'll remove the comment for now.